scripts/common.py

```diff
@@ -127,6 +131,8 @@
 # Directory for storing/fetching data related to the Firebase emulator.
 FIREBASE_EMULATOR_CACHE_DIR = (
     os.path.join(CURR_DIR, os.pardir, 'firebase_emulator_cache'))
+sys.path.insert(0, PY_GITHUB_PATH)
+import github # isort:skip  pylint: disable=wrong-import-position
```

Comment:
Why is this added here?

Reply:
To type annotate [this](https://github.com/oppia/oppia/blob/46ee2fa3747afdba675cb26790aca1cbfc923468/scripts/common.py#L551) one we need the github module and for that, I tried to import github module here like this

Reply:
Hmm, can you only import it in `if MYPY:` block as we do in other places with other imports?

Reply:
Yeah sure. I am not adding type annotation in common.py in this PR. For that file, I have created [this](https://github.com/oppia/oppia/pull/15258/) one, just to debug some errors caused in previous tests. There I have added this if block. Thanks